dvc/command/live.py

```diff
@@ -16,7 +17,8 @@ def _run(self, target, revs=None):
             html_path = Path.cwd() / (self.args.target + "_html")
             from dvc.render.utils import render
 
-            index_path = render(self.repo, plots, metrics, html_path)
+            renderers = match_renderers(plots, self.repo.plots.templates)
```

Comment:
@pared, would it be possible to push all of this to renderer/plots? Something similar to jinja for example:
```python
env = Environment(loader=PackageLoader("yourapp"))
template = env.get_template("default")
print(template.render(...)
```

I am not very aware of this subsystem, but it may be much convenient instead of using `match_renderer` and `render`?

Reply:
@skshetry you mean, to use Jinia as a templating engine for our `Vega` and `Image` renderers?

Reply:
And I am not sure what would you like to move to `renderer/plots`? Like unifying matching renderers and HTML generation? I intentionally separated it to have `match_renderers` called in `command/plots` only once. Before, we had a separate functions to match renderers for `show-vega` and HTML generation.